Lecture4.tex

```diff
@@ -16,11 +16,11 @@ \section{Markov Chains: A Reminder}
 We focus on time-homogeneous Markov chains, where
 \[{\cal P}({X_{t + 1}} = j|{X_t} = i) = {\cal P}({X_1} = j|{X_0} = i) \buildrel \Delta \over = {p_{ij}}.\]
 The ${p_{ij}}$'s  are the transition probabilities, which satisfy
-${p_{ij}} \ge 0,\;\;\;\sum\nolimits_{i \in X} {{p_{ij}} = 1\;\;\forall j} $.
+${p_{ij}} \ge 0,\;\;\;\sum\nolimits_{j \in X} {{p_{ij}} = 1\;\;\forall i} $.
 The matrix $P = ({p_{ij}})$ is the transition matrix.
 
-Given the initial distribution ${p_0}$ of ${X_0}$, namely $p({X_0} = i) = {p_0}(i)$, we obtain the finite-dimensional distributions:
-\[{\cal P}({X_0} = {i_0}, \ldots ,{X_t} = {i_t}) = {p_0}(i){p_{{i_0}{i_1}}} \cdot  \ldots  \cdot {p_{{i_{t - 1}}{i_t}}}.\]
+Given the initial distribution ${p_0}$ of ${X_0}$, namely $p({X_0} = i_0) = {p_0}(i_0)$, we obtain the finite-dimensional distributions:
```

Comment:
i is OK here (general element in the vector). Only in the next line it should be i_0